pytorch_lightning/profiler/pytorch.py

```diff
@@ -69,7 +69,8 @@ def __init__(self, model: nn.Module) -> None:
         self._handles: Dict[str, List["RemovableHandle"]] = {}
 
     def _start_recording_forward(self, _: nn.Module, input: Tensor, record_name: str) -> Tensor:
-        record = record_function(record_name)
+        # Add [pl][module] in name for pytorch profiler to recognize
+        record = record_function("[pl][module]" + record_name)
```

Comment:
Note for below. Would be great to add decoration for the backward pass too. As we can access the loss, we can record backward.

Reply:
Good suggestion! Maybe we can add it in another pr.